### PR TITLE
LPS-142295 Prepopulate default custom element data

### PR DESCRIPTION
**Pull request by izaera**
By default, we will point to Vanilla Counter custom element, hosted in
liferay-frontend-project's GitHub pages site.

### CODE REVIEW

**Comment by izaera** [comment]
This was approved in #1654 already, but we are sending it detached to wait a bit to forward it.

**Review by nhpatt — APPROVED** [review]
Waiting...

File: modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/display/context/EditRemoteAppEntryDisplayContext.java [hunk 1]
```diff
@@ -100,15 +106,24 @@ public String getCmd() {
 			_portletRequest, "customElementURLs", customElementURLs);
 
 		if (customElementURLs.length == 0) {
-			customElementURLs = new String[1];
+			customElementURLs = new String[] {
+				"https://liferay.github.io/liferay-frontend-projects" +
+					"/vanilla-counter/index.js"
```
Comment (diegonvs) [inline modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/display/context/EditRemoteAppEntryDisplayContext.java:111]: When going to: `https://liferay.github.io/liferay-frontend-projects/vanilla-counter/index.js` it returns 404. I think it's not published yet

We could create a repo that publishes as a npm package and bring those files from unpkg

**Comment by diegonvs** [comment]
ci:test:relevant

File: modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/display/context/EditRemoteAppEntryDisplayContext.java (hunk 1 above)
Comment (izaera) [reply modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/display/context/EditRemoteAppEntryDisplayContext.java:111]: We are waiting for IT to publish it (we don't have permissions). But once it is published we will just need to push changes to the `gh-pages` branch of `liferay-frontend-projects` (easier than using npm, unpkg, etc...).

Also, Brian wanted it to be under a Liferay domain, so we will publish it in GitHub, then point a liferay.com DNS entry to the site. Most probably.

**Comment by izaera** [comment]
After finding out what the cause of the CI error was, @magjed4289 will take care of it :+1: 

We are also missing the GH pages for repo [liferay-frontend-projects](https://github.com/liferay/liferay-frontend-projects/) publication, which is still pending work from LRIT that @brunofernandezg requested.

**Comment by magjed4289** [comment]
Just started reviewing :)

**Comment by magjed4289** [comment]
ci:bundle

**Comment by magjed4289** [comment]
ci:test:bundle

**Comment by izaera** [comment]
will resend with a newer approach

**Pull request closed without merging**